### PR TITLE
Markup - Create components and pages related to the creation of listings (accommodation)

### PR DESCRIPTION
**Pull request by rianfrancis**
## Changes Made

- Created new listing form

## Purpose

As a host, I want to create a new listing by providing details about my space so that guests can find my offering.

## Related Task/Issue

Link to the related task or issue that prompted these changes.

- Slack Link: https://sun-philippine.slack.com/archives/C06JZLS2W5S/p1709803618997719
- Task Link: https://framgiaph.backlog.com/view/SBNB-56#comment-179773120

## Screenshots

Include screenshots to visually demonstrate the changes made by this pull request.

image
_Caption for Screenshot 1_

image
_Caption for Screenshot 2_

image
_Caption for Screenshot 3_

## New Dependencies (if applicable)

- N/A

## Additional Information (if applicable)

image
Upload photo is subject to change due to backend implementation

### CODE REVIEW

**Review by johnvergilaparriSA — APPROVED** [review]
I think this is all good

File: frontend/src/app/components/NewListingForm.tsx [hunk 1]
```diff
@@ -12,134 +21,109 @@ const NewListingForm: React.FC = () => {
...
-                            <span>{amenity.name}</span>
-                        </label>
+
+            <div className="mb-10 mt-5 rounded-lg border-[1.3px] border-solid border-[color:var(--Blues-Gray2,#B8BBC2)] p-10">
+                <div className="grid grid-cols-2">
+                    <div>
+                        <div className="mb-5 w-full text-left text-left text-sm font-semibold leading-5 text-black max-md:max-w-full">
+                            Amenities
+                        </div>
+                        <Listboxx />
```
Comment (brettjosef) [inline frontend/src/app/components/NewListingForm.tsx:109]: You can name this as ListBoxComponent

File: frontend/src/app/components/NewListingForm.tsx [hunk 2]
```diff
@@ -12,134 +21,109 @@ const NewListingForm: React.FC = () => {
...
-            </div>
-            <div>
-                {details.map((detail, index) => (
-                    <div
-                        key={index}
-                        className="mt-2 flex w-full justify-between rounded-lg border-[1.3px] border-solid border-[color:var(--Blues-Gray2,#B8BBC2)] bg-white px-5 py-3 text-base leading-5 tracking-wide text-zinc-500"
-                    >
-                        <div className="flex-auto">{detail.title}</div>
-                        {/* Wa ko kahibao unsaon pag add ug counter :( */}
+                    <div>
```
Comment (brettjosef) [inline frontend/src/app/components/NewListingForm.tsx:111]: This as well

File: frontend/src/app/components/NewListingForm.tsx [hunk 3]
```diff
@@ -12,134 +21,109 @@ const NewListingForm: React.FC = () => {
...
-                    >
-                        <label className="flex items-center">
-                            <input type="checkbox" className="mr-2" />
-                            <span>{amenity.name}</span>
-                        </label>
+
+            <div className="mb-10 mt-5 rounded-lg border-[1.3px] border-solid border-[color:var(--Blues-Gray2,#B8BBC2)] p-10">
+                <div className="grid grid-cols-2">
+                    <div>
+                        <div className="mb-5 w-full text-left text-left text-sm font-semibold leading-5 text-black max-md:max-w-full">
```
Comment (brettjosef) [inline frontend/src/app/components/NewListingForm.tsx:106]: You can add this div to the ListboxComponent

File: frontend/src/app/components/NewListingForm.tsx [hunk 4]
```diff
@@ -12,134 +21,109 @@ const NewListingForm: React.FC = () => {
...
             </div>
-            <select className="mt-8 w-full items-start justify-center whitespace-nowrap rounded-2xl border-2 border-solid border-[color:var(--Semantic-light-layout-content3,#E4E4E7)] bg-white bg-opacity-0 py-5 pl-3.5 pr-16 text-base leading-6 text-zinc-500 shadow-sm max-md:max-w-full max-md:pr-5">
-                <option value="">Select accommodation type</option>
-                <option value="apartment">Apartment</option>
-                <option value="house">House</option>
-                <option value="villa">Villa</option>
-                <option value="cottage">Cottage</option>
-            </select>
-            <div className="mt-10 w-full text-center text-sm font-semibold leading-5 text-black max-md:max-w-full">
+            <Select
```
Comment (brettjosef) [inline frontend/src/app/components/NewListingForm.tsx:39]: Transfer to a new component as well

**Review by brettjosef — APPROVED** [review]
This looks good

**Pull request merged**